### PR TITLE
[WIP] Add EasyConfig files for lastest QEMU

### PR DESCRIPTION
**Pull request by besserox**
Based on the work from @hcartiaux in PR https://github.com/hpcugent/easybuild-easyconfigs/pull/1844

Since we don't have intel/2016a installed yet, we could not test for this toolchain. But we would like to have it since we plan to use it in the future. I guess you will test if it builds anyway.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2044.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/788f8488ea7f203944e8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2721.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3c18b3a0980f4e86c0f5 for a full test report.

**Comment by boegel** [comment]
@besserox: missing deps:

```
ERROR: glib-2.22 gthread-2.0 is required to compile QEMU
```

File: easybuild/easyconfigs/q/QEMU/QEMU-2.5.0-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    ('libattr1-dev', 'libattr-devel'),
+    ('libcap-dev', 'libcap-devel'),
+    ('libaio-dev', 'libaio-devel'),
+    ('texinfo'),
+    ('libglib2.0-dev', 'glib2-devel')
+]
+
+dependencies = [
+                ('zlib', '1.2.8'),
+               ]
```
Comment (boegel) [inline easybuild/easyconfigs/q/QEMU/QEMU-2.5.0-foss-2016a.eb]: please fix indent (no tabs please):

```python
dependencies = [
    ('zlib', '1.2.8'),
]
```

File: easybuild/easyconfigs/q/QEMU/QEMU-2.5.0-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+sanity_check_paths = {
+                       'files': ['bin/qemu-img', 'bin/qemu-io'],
+                       'dirs': []
+                     }
```
Comment (boegel) [inline easybuild/easyconfigs/q/QEMU/QEMU-2.5.0-foss-2016a.eb]: please fix indent:

```python
sanity_check_paths = {
    'files': ['bin/qemu-img', 'bin/qemu-io'],
    'dirs': [],
}
```

File: easybuild/easyconfigs/q/QEMU/QEMU-2.5.0-foss-2016a.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://wiki.qemu-project.org/download/']
+configopts = "--disable-rdma --enable-kvm --enable-docs --disable-gtk --enable-linux-aio --enable-virtfs"
+
+osdependencies = [
+    ('libattr1-dev', 'libattr-devel'),
+    ('libcap-dev', 'libcap-devel'),
+    ('libaio-dev', 'libaio-devel'),
+    ('texinfo'),
+    ('libglib2.0-dev', 'glib2-devel')
```
Comment (boegel) [inline easybuild/easyconfigs/q/QEMU/QEMU-2.5.0-foss-2016a.eb]: several of these can be resolved through EasyBuild instead, which would significantly help imho...

**Comment by besserox** [comment]
OK. Thanks for testing. I'll try to fix this.

**Comment by migueldiascosta** [comment]
@besserox, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**